src/components/navbar.tsx

```diff
@@ -31,42 +32,65 @@ const Navbar = () => {
     { name: dict.navbar.projects, path: "/projects" },
   ];
 
+  const isMemberOrCollaborator =
+    pathname === "/about/become-a-member" ||
+    pathname === "/about/become-a-collaborator";
+
+  const navbarBackgroundColor =
+    pathname === "/about/become-a-member"
+      ? "bg-primary"
+      : pathname === "/about/become-a-collaborator"
+        ? "bg-blue"
+        : "bg-background";
+  const linkColor = isMemberOrCollaborator ? "text-frost" : "text-gray";
```

Comment:
```suggestion
  const linkColor = isMemberOrCollaborator ? "text-white/50" : "text-gray";
```

Refer to my other comment.